### PR TITLE
implemented taxi start

### PR DESCRIPTION
**Pull request by mariiahorbova**
drivers
manufacturers
cars

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+# Cython debug symbols
+cython_debug/
+
+# PyCharm
+#  JetBrains specific template is maintained in a separate JetBrains.gitignore that can
+#  be found at https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore
+#  and can be added to the global gitignore or merged into this file.  For a more nuclear
+#  option (not recommended) you can uncomment the following to ignore the entire idea folder.
+.idea/
```
Comment (YuriiKindrat) [inline .gitignore:162]: no blank line at the end of file ((

File: taxi/models.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+    def __str__(self) -> str:
+        return f"Name: {self.name} Country: {self.country}"
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=63)
+    manufacturer = models.ForeignKey(
+        to=Manufacturer,
+        on_delete=models.DO_NOTHING,
```
Comment (YuriiKindrat) [inline taxi/models.py:26]: why did u choose do nothing option ?

File: taxi/models.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        related_name="cars"
+    )
+    driver = models.ManyToManyField(to=settings.AUTH_USER_MODEL)
```
Comment (YuriiKindrat) [inline taxi/models.py:29]: what about related name ?

**Review by YuriiKindrat — CHANGES_REQUESTED** [review]

File: taxi/models.py (hunk 2 above)
Comment (mariiahorbova) [reply taxi/models.py:26]: because even if we manufacturer stops existing, cars will still be there

File: taxi_service/settings.py [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+
+INSTALLED_APPS = [
+    "django.contrib.admin",
+    "django.contrib.auth",
+    "django.contrib.contenttypes",
+    "django.contrib.sessions",
+    "django.contrib.messages",
+    "django.contrib.staticfiles",
+    "taxi",
+]
```
Comment (Oleg-s-s) [inline taxi_service/settings.py:43]:
```suggestion
INSTALLED_APPS = [
    "django.contrib.admin",
    "django.contrib.auth",
    "django.contrib.contenttypes",
    "django.contrib.sessions",
    "django.contrib.messages",
    "django.contrib.staticfiles",
    
    "taxi",
]
```

**Review by Oleg-s-s — APPROVED** [review]
Its better to clean your project from unnecessary docstrings and comments

**Pull request closed without merging**